resources/js/messages.ts

```diff
@@ -11,7 +11,7 @@ const messages = {
     pt: {
         homePage: {
             aboutUsText1:
-                "A Semana de Informática (SINF), organizada pelo Núcleo de Informática da Associação de Estudantes da Faculdade de Engenharia da Universidade do Porto (NIAEFEUP), foi criada com o intuito de permitir aos estudantes, independentemente do curso, desenvolver as suas capacidades nas diversas áreas da Informática, promovendo a sua interação com o mundo empresarial através de eventos sociais.",
+                "A Semana de Informática (SINF), organizada pelo Núcleo de Informática da Associação de Estudantes da Faculdade de Engenharia da Universidade do Porto (NIAEFEUP), foi criada com o intuito de permitir aos participantes, independentemente do curso, desenvolver as suas capacidades nas diversas áreas da Informática, promovendo a sua interação com o mundo empresarial através de eventos sociais.",
```

Comment:
```suggestion
                "A Semana de Informática (SINF), organizada pelo Núcleo de Informática da Associação de Estudantes da Faculdade de Engenharia da Universidade do Porto (NIAEFEUP), foi criada com o intuito de permitir aos participantes desenvolver as suas capacidades nas diversas áreas da Informática, promovendo a sua interação com o mundo empresarial através de eventos sociais.",
```
Also, if we don't want to imply that this event is only for students, we should remove the segments from the about us section that say "no matter the course".

Reply:
Not sure what everyone else thinks about this, though

Reply:
I forgot to ask about that, just changing "estudantes" to "partipantes" sounds weird. I'm going to change that and also the name of one of the migrations